payment-management-service/payment-management-client/src/main/resources/api.yaml

```diff
@@ -0,0 +1,165 @@
+          description: Authorization token
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PaymentDto"
+      responses:
+        "200":
+          description: Returns created Payment
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PaymentDto"
+
+    put:
+      tags:
+        - Payments
+      description: Updates payment
+      operationId: updatePayment
+      parameters:
+        - in: header
+          name: Authorization
+          schema:
+            type: string
+          required: true
+          description: Authorization token
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PaymentDto"
+      responses:
+        "200":
+          description: Returns updated payment
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PaymentDto"
+
+  /{id}:
+    get:
+      tags:
+        - Payments
+      description: Get payment by id
+      operationId: getPaymentById
+      parameters:
+        - in: header
+          name: Authorization
+          schema:
+            type: string
+          required: true
+          description: Authorization token
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+          description: Payment id
+      responses:
+        "200":
+          description: Returns specific payment
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PaymentDto"
+    delete:
+      tags:
+        - Payments
+      description: Delete payment by id
+      operationId: deletePaymentById
+      parameters:
+        - in: header
+          name: Authorization
+          schema:
+            type: string
+          required: true
+          description: Authorization token
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+          description: Payment id
+      responses:
+        "200":
+          description: Payment deleted
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PaymentDto"
+
+components:
+  schemas:
+
+    PaymentStatus:
+      type: string
+      enum:
+        - Pending
+        - Completed
+        - Cancelled
+
+
+    PaymentDto:
+      description: Information about payment
+      type: object
+      properties:
+        id:
+          type: integer
+        paymentProvider:
+          type: string
+        date:
+          type: string
+        price:
+          type: float
+        paymentStatus:
+          $ref: "#/components/schemas/PaymentStatus"
+      required:
+        - price
+        - paymentProvider

```

Comment:
Add property 'orderId', because once service accept payment, it should notify order management service, that payment has been completed